db/migrate/20160603182320_add_en_passant_file_to_games.rb

```diff
@@ -0,0 +1,5 @@
+class AddEnPassantFileToGames < ActiveRecord::Migration
+  def change
+    add_column :games, :en_passant_file, :integer
```

Comment:
Ah, I see that [it is](https://github.com/letmecheck/app/pull/56/commits/d4fd906f54db0089d127b08b29fb66777660a6e1#diff-1b541d09ba44dabaa60af6e62f436119R26). I think I would have expected a boolean (e.g., `en_passant_eligible`) here rather than an integer. Is the current `en_passant_file` (the integer, that is) used for anything during gameplay?